quote-book/synchronized/src/lib.rs

```diff
@@ -1,111 +1,392 @@
 // Copyright (c) 2023 MobileCoin Inc.
-
-use deqs_quote_book_api::{Error, Pair, Quote, QuoteBook, QuoteId};
+// Copyright (c) 2023 MobileCoin Inc.
+use deqs_quote_book_api::{Error as QuoteBookError, Pair, Quote, QuoteBook, QuoteId};
```

Comment:
Any reason for this change?

Reply:
Not particularly. I can back it out. It was to help me with some confusion I was having with the different error types when I had 3 of them, but having removed the backoff crate, this should be fine to go back.